app/views/layouts/_nav_menu.html.erb

```diff
@@ -1,64 +1,32 @@
-<div class="top-bar--menu-dropdown">
+<div class="crayons-dropdown p-0 top-bar__menu__dropdown">
   <% if user_signed_in? %>
-    <div id="user-profile-link-placeholder">
-      <div class="option">
-        ...
-      </div>
+    <div id="user-profile-link-placeholder" class="border-0 border-b-1 border-solid border-base-20 p-1">
```

Comment:
It seems like "grouping links" can be potentially useful as global component (or part of it) and I'm gonna create one. But until then, I'm gonna use this utility-first classes over there.